src/spec/timepicker/timepicker-controls.util.spec.ts

```diff
@@ -1,22 +1,16 @@
-import {
-  canChangeHours,
-  canChangeMinutes,
-  canChangeSeconds,
-  canChangeValue,
-  getControlsValue,
-  timepickerControls
-} from '../../timepicker/timepicker-controls.util';
 import { TimeChangeEvent, TimepickerComponentState, TimepickerControls } from '../../timepicker/timepicker.models';
+import * as timepickerControlsUtil from '../../timepicker/timepicker-controls.util';
```

Comment:
please avoid usage of wildcard imports

Reply:
@valorkin what if we use **all** functions from this file? Is there any sense in importing all the functions in this case?

Reply:
wildcard imports is a bad practice
makes harder to split code later
or can fail IDE helpers like methods renames, etc.